gui/main.py

```diff
@@ -105,6 +105,26 @@ def dialog_is_yes(dialog):
     return dialog.run() == Gtk.ResponseType.OK
 
 
+class ActionName(str):
+    def short_name(self):
+        return self.split('.')[-1]
+
+
+class app(object):
```

Comment:
I wanted to use enum.Enum at first, but it does weird things… apparently the instance isn't really a string (or the class you made it), so I can't really use my `ActionName` trick, and Gtk isn't happy:

```
Traceback (most recent call last):
  File "/media/data/Hypra/dev/oomox/gui/main.py", line 398, in do_activate
    self.window = AppWindow(application=self)
  File "/media/data/Hypra/dev/oomox/gui/main.py", line 348, in __init__
    self._init_window()
  File "/media/data/Hypra/dev/oomox/gui/main.py", line 314, in _init_window
    self._init_headerbar()
  File "/media/data/Hypra/dev/oomox/gui/main.py", line 259, in _init_headerbar
    clone_button.set_action_name(win.clone)
TypeError: Must be string, not win
```